examples/manipulating_visualizing/plot_dim_plotting.py

```diff
@@ -0,0 +1,51 @@
+"""
+Controling the contrast of the background when plotting
+=========================================================
+
+The `dim` argument controls the contrast of the background.
```

Comment:
> perhaps also mention here for what membrs of the plot_* function stack this
> holds/is useful.

That' not completely true, as it doesn't apply to all functions. I have
added a comment on plot_roi, though.